src/PayfortIntegration.php

```diff
@@ -57,6 +59,33 @@ public function setMerchant(array $merchant): self
         return $this;
     }
 
+
+    public function setApplePayParams(array $params = []): self
```

Comment:
it's better to accept each needed attribute as a separate param to make sure the user passes them 

```suggestion
    public function setApplePayParams(array $data, array $header, array $payment_method): self
```

and they shouldn't have default values to make sure the user passes them